src/index.js

```diff
@@ -1,31 +1,34 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import registerServiceWorker from './registerServiceWorker';
-import createStore from './Services/Store';
-import createHistory from 'history/createBrowserHistory';
-import { syncHistoryWithStore } from 'react-router-redux';
-import { Provider } from 'react-redux';
-import { BrowserRouter as Router } from 'react-router-dom';
-import publicRoutes from './routes';
-import axe from 'react-axe';
-import App from './App';
+// import React from 'react';
+// import ReactDOM from 'react-dom';
+// import registerServiceWorker from './registerServiceWorker';
+// import createStore from './Services/Store';
+// import createHistory from 'history/createBrowserHistory';
+// import { syncHistoryWithStore } from 'react-router-redux';
+// import { Provider } from 'react-redux';
+// import { BrowserRouter as Router } from 'react-router-dom';
+// import publicRoutes from './routes';
+// import axe from 'react-axe';
+// import App from './App';
+import Table from './Components/Table';
+export default { Table };
```

Comment:
I thought `default` broke this?

Reply:
you're right.. I have more I forgot to commit.  Adding it now.